modules/dnn/misc/python/test/test_dnn.py

```diff
@@ -305,5 +311,32 @@ def forward(self, inputs):
 
         cv.dnn_unregisterLayer('CropCaffe')
 
+    def test_lstm(self):
+        model = self.find_dnn_file('dnn/onnx/models/hidden_lstm.onnx')
+        input_file = self.find_dnn_file('dnn/onnx/data/input_hidden_lstm.npy')
+        output_file = self.find_dnn_file('dnn/onnx/data/output_hidden_lstm.npy')
+        if model is None:
+            raise unittest.SkipTest("Missing DNN test files (dnn/onnx/models/hidden_lstm.onnx). "
+                                    "Verify OPENCV_DNN_TEST_DATA_PATH configuration parameter.")
+        if input_file is None or output_file is None:
+            raise unittest.SkipTest("Missing DNN test files (dnn/onnx/data/{input/output}_hidden_lstm.npy). "
+                                    "Verify OPENCV_DNN_TEST_DATA_PATH configuration parameter.")
+
+        net = cv.dnn.readNet(model)
+        input = np.load(input_file)
+        # we have to expand the shape of input tensor because Python bindings cut 3D tensors to 2D
+        input = np.expand_dims(input, axis=3)
```

Comment:
I'm OK to merge PR with changed name of the test: `test_lstm` -> `test_input_3d`

It makes sense to add here a comment with link on https://github.com/opencv/opencv/issues/19091 issue
Requirement of `expand_dims` statement would be removed in the future.

Reply:
Thanks. Updated.